src/Shared/CodeAnnotations.cs

```diff
@@ -1,6 +1,3 @@
-// Copyright (c) .NET Foundation. All rights reserved.
-// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
-
```

Comment:
Should this header be removed?

Reply:
I assumed that was an unintended artifact of some earlier copy/paste, but I guess it could be needed... 

@roji Thoughts?